.circleci/config.yml

```diff
@@ -65,7 +65,9 @@ references:
   install_js_packages: &install_js_packages
     run:
       name: Install Yarn packages
-      command: yarn --frozen-lockfile
+      command: |
+        npm install
```

Comment:
We use yarn so it'd be `yarn install` . Just curious to know what issue this npm install command resolved?

Reply:
Ah gotya 👍 . 
How about trying the below (i'm still learning front-end stuff but the below should help)
```suggestion
    yarn install --frozen-lockfile
```

Reply:
Cheers. I'll try removing the commit from this branch anyway as I don't think we need it for the ruby 2.6  deploys on circle and I'll play with it on #1292 :+1: